network.js

```diff
@@ -5,12 +5,12 @@ export default {
     'Cosmos is a decentralized network of independent parallel blockchains, each powered by BFT consensus algorithms like Tendermint consensus.',
   logo: `logo.svg`,
   website: 'https://cosmos.network',
-  apiURL: 'https://lcd.nylira.net',
-  stakingDenom: 'ATOM',
+  apiURL: 'https://api.allorigins.win/raw?url=http://34.123.30.100:1317',
```

Comment:
@Bitcoinera see handling for CORS

Reply:
oh, so easy? I was changing nuxt config like this:

```
  axios: {
    progress: false,
    retry: { retries: 3 },
    proxy: true,
  },

  proxy: {
    '/': 'https://cors-anywhere.herokuapp.com/',
  },
```

But it wasn't working :shrug: Although according to the documentation this also should work